db/versions/0113.yml

```diff
@@ -0,0 +1,80 @@
+version: 113
+description: get correct number of claimed tasks for a task queue
+methods:
+  queue_claimed_tasks_count:
+    description: |
+      Count the number of claimed tasks for given task queue.
+      Because queue_claimed_tasks table might have several records for the same task-run
+      when the task is being reclaimed, we count distinct (task_id, run_id)
+    mode: read
+    serviceName: queue
+    args: task_queue_id_in text
+    returns: integer
+    body: |-
+      begin
+        return (
+          select count(distinct (task_id, run_id))
+          from queue_claimed_tasks
+          where task_queue_id = task_queue_id_in
+            and taken_until > now()
+        );
+      end
+
+  queue_worker_stats:
```

Comment:
this is the same query as in v112, but just uses `DISTINCT (task,run)` instead of string concat